src/stactools/palsar2_scansar/constants.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime
+from typing import Any, Dict, Optional
+
+import pystac
+from pystac import Extent, Link, Provider
+from pystac import ProviderRole as PR
+from pystac import SpatialExtent, TemporalExtent
+from pystac.extensions import sar
+from pystac.extensions.eo import Band
+from pystac.extensions.item_assets import AssetDefinition
+from pystac.extensions.raster import DataType
+from pystac.utils import str_to_datetime
+
+# Time must be in UTC
+SCANSAR_COLLECTION_START: Optional[datetime] = str_to_datetime("2014-08-04T00:00:00Z")
+SCANSAR_TEMPORAL_EXTENT = TemporalExtent([[SCANSAR_COLLECTION_START, None]])
+
+# TODO: update to match bucket
+SCANSAR_SPATIAL_EXTENT = SpatialExtent([-180.0, -56, 180.0, 85.0])
+
+SCANSAR_EXTENT = Extent(SCANSAR_SPATIAL_EXTENT, SCANSAR_TEMPORAL_EXTENT)
+
+SCANSAR_PALSAR_PROVIDERS = [
+    Provider(
+        "Japan Aerospace Exploration Agency",
+        roles=[PR.PRODUCER, PR.PROCESSOR, PR.LICENSOR],
+        url="https://www.eorc.jaxa.jp/ALOS/en/dataset/fnf_e.htm",
+    ),
+    Provider(
+        "Japan Aerospace Exploration Agency",
+        roles=[PR.HOST],
+        url="https://registry.opendata.aws/jaxa-alos-palsar2-scansar/",
+    ),
+]
+SCANSAR_DESCRIPTION = """The new PALSAR-2 ScanSAR products have been developed to conform with the CEOS-ARD format specifications for Normalised Radar Backscatter (CARD4L NRB). The NRB products have been subject to ortho-rectification and Radiometric Terrain Correction (RTC), and are provided in the Gamma-0 backscatter convention, which mitigates the variations from diverse observation geometries. The products include HH and HV polarisation backscatter, local incidence angle image and data mask."""  # noqa: E501
+
+SCANSAR_REVISION = "E"
+
+SCANSAR_LINKS = [
+    Link(
+        "documentation",
+        ("https://www.eorc.jaxa.jp/ALOS/en/dataset/palsar2_l22_e.htm"),
+        "application/pdf",
+        SCANSAR_DESCRIPTION,
```

Comment:
The title for the documentation link is the entire SCANSAR description. Should we slim it down?